packages/core/src/telemetry/emitter.ts

```diff
@@ -123,9 +147,15 @@ export const telemetryEmitter = async ({ dataDir, version, debug }: {
     setProps: (props: TelemetryProperties) => {
       Object.assign(commonProperties, props)
     },
-    flush: () => {
-      throttledFlush.cancel()
-      return flush()
+    // For making sure we're not keeping the process running due to debounce.
+    // Flush is not called as there could be other captures afterwards such as exit event.
+    release: () => {
```

Comment:
I think a more idiomatic name is `unref` - see [timeout](https://nodejs.org/api/timers.html#timeoutunref), [worker thread](https://nodejs.org/api/worker_threads.html#workerunref), [socket](https://nodejs.org/api/dgram.html#socketunref) (there's more).